hpy/debug/leakdetector.py

```diff
@@ -1,16 +1,18 @@
 from hpy.universal import _debug
 
-class HPyError(Exception):
+class HPyDebugError(Exception):
     pass
 
-class HPyLeak(HPyError):
+class HPyLeakError(HPyDebugError):
     def __init__(self, leaks):
         super().__init__()
         self.leaks = leaks
 
     def __str__(self):
         lines = []
-        lines.append('%s handles have not been closed properly:' % len(self.leaks))
+        n = len(self.leaks)
+        s = 's' if n > 1 else ''
```

Comment:
This should be `'s' if n != 1 ...` since 0 handles should display `0 unclosed handles`.
```suggestion
        s = 's' if n != 1 else ''
```

Reply:
maybe, although you should never get this if you have 0 unclosed handles 😅. But I suppose it doesn't hurt